pkg/apis/psmdb/v1/perconaservermongodbrestore_types.go

```diff
@@ -177,6 +177,8 @@ func (t *PITRestoreDate) MarshalJSON() ([]byte, error) {
 	return json.Marshal(t.Time.Format("2006-01-02 15:04:05"))
 }
 
+// +kubebuilder:validation:XValidation:rule="self.type != 'date' || (has(self.date) && self.date.matches('^[0-9]{4}-[0-9]{2}-[0-9]{2} [0-9]{2}:[0-9]{2}:[0-9]{2}$'))",message="Time should be in format YYYY-MM-DD HH:MM:SS"
```

Comment:
We can create a small test for ensuring that in our controller test file. Mainly checking that with the right configuration is passes and with wrong the expected error message is triggered.